notebooks/lesson07_classification.ipynb

```diff
@@ -0,0 +1,3932 @@
+{
```

Comment:
<p>This assumes that you already know that " " is in the dataset. Maybe it would be a good opportunity to show the students how they could investigate/debug the dataset (maybe with a value counter?).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>good idea - done!</p>